lib/Command/Configure/Check.php

```diff
@@ -61,13 +62,27 @@ protected function configure(): void {
 				InputOption::VALUE_NONE,
 				'Check requirements to use root certificate'
 			);
+		if ($this->pagePreviewAsImage) {
+			$this
+				->addOption('preview',
+					'p',
+					InputOption::VALUE_NONE,
+					'Check requirements to generate image preview'
+				);
+		}
 	}
 
 	protected function execute(InputInterface $input, OutputInterface $output): int {
-		$preview = $input->getOption('preview');
-		$sign = $input->getOption('sign');
-		$certificate = $input->getOption('certificate');
-		$all = (!$preview && !$sign && !$certificate);
+		if ($this->pagePreviewAsImage) {
+			$preview = $input->getOption('preview');
+			$sign = $input->getOption('sign');
+			$certificate = $input->getOption('certificate');
+			$all = (!$preview && !$sign && !$certificate);
+		} else {
+			$sign = $input->getOption('sign');
+			$certificate = $input->getOption('certificate');
+			$all = (!$sign && !$certificate);
+		}
```

Comment:
The code for each condition is so similar - just an aleatory comment.

Reply:
Yes, very ugly. I made a small improvement, would you check again?